misk-hibernate/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -76,6 +77,20 @@ abstract class TransacterTest {
         assertThat(query.uniqueResult(session)!!.actor?.name).isEqualTo("Jeff Goldblum")
       }
     }
+    readerTransacter.transaction { session ->
```

Comment:
Could you add a test that you try to do an update in this session and should throw some sort of ReadOnly exception?

Reply:
added. I'm reusing the `readOnly` behavior in the transacter.